mkosi/config.py

```diff
@@ -2879,8 +2890,10 @@ def load_environment(args: argparse.Namespace) -> dict[str, str]:
     if dnf := os.getenv("MKOSI_DNF"):
         env["MKOSI_DNF"] = dnf
 
-    for s in args.environment:
+    entries = [x for y in (f.read_text().strip().splitlines() for f in args.environment_files) for x in y]
```

Comment:
This still seems overly complex, what about the following:

```
entries = [l for f in args.environment_files for l in f.read_text().strip().splitlines()]
```

Reply:
oo this works, thanks